message/typeset.go

```diff
@@ -30,6 +30,65 @@ func TypesOf(messages ...dogma.Message) TypeSet {
 	return s
 }
 
+// IntersectionT returns the intersection of the given collections.
+//
+// That is, it returns a set containing only those types that are present in all
```

Comment:
Should it mention that if the given collections are not intersecting, then an empty set is returned?

Reply:
I didn't think this needed to be explained separately as it's not a special case.